features.md

```diff
@@ -159,17 +159,17 @@ On the other hand, any of the following actors cannot single-handedly decrypt a
 
 ### Functional perimeter
 
-A *user* can share data with another *user* who is not registered on the *application server* or on the *Trustchain* yet. Such a recipient is identified by their email address.
+A *user* can share data with another *user* who is not registered on the *application server* or on the *Trustchain* yet. Such a recipient is identified by their email address or phone number.
```

Comment:
At some point we should say `the target value of their provisional identity` or something no ?

Reply:
Applied.